apps/extension/src/utils/download-proving-keys.mjs

```diff
@@ -0,0 +1,44 @@
+import fetch from 'node-fetch';
+import * as fs from 'fs';
+import * as path from 'path';
+
+const githubSourceDir =
+  'https://github.com/penumbra-zone/penumbra/raw/main/crates/crypto/proof-params/src/gen/';
+
+const provingKeyFiles = [
+  { file: 'spend_pk.bin' },
+  { file: 'output_pk.bin' },
+  { file: 'swap_pk.bin' },
+  { file: 'swapclaim_pk.bin' },
+  { file: 'nullifier_derivation_pk.bin' },
+  { file: 'delegator_vote_pk.bin' },
+  { file: 'undelegateclaim_pk.bin' },
+];
```

Comment:
Could we re-use one array instead of having this in two places? Perhaps we can move https://github.com/penumbra-zone/web/pull/283/files#diff-792d69d65ae8bb0bcbeeec13edcf4c7aded7bf621432261ac326705f69d88d29L34-L42 to the types pack and both consumers of this can share it.

Reply:
that's a good idea, changed